### ISSUE TITLE
Inspiration: Chip Inventory Graphic Glitches When Turning Right or Left

### ISSUE THREAD

**Issue by rparnas** [elasota/scummvm#147]
Acquire the chip, then turn left or right and the inventory graphic glitches.

Not present in retail

Tested against:
* Win retail on Win98 SE
* [SCUMM: (LOOM/EGA) - fix bug 13763 (Inaccurate text position) · scummvm/scummvm@b4185bd](https://github.com/scummvm/scummvm/commit/b4185bdf60f8f28afe57b192f7073135e0eb2cc8)

**Comment by elasota** [comment]
Yeah this affects a lot of things with the widescreen mod in particular.  It looks like objects in the shared scene aren't affected by push transitions, though haven't figured out how that's supposed to work with respect to layering.  It's actually weirdly glitchy in mTropolis Player in some places like the Junkyard.
